view/frontend/layout/default.xml

```diff
@@ -11,6 +11,11 @@
                     <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
                 </arguments>
             </block>
+            <block name="extend.aftermarket-product-protection-modal-offer" ifconfig="extend/product_protection/enable" template="Extend_Integration::catalog/product/view/aftermarket-product-protection-modal-offer.phtml">
+                <arguments>
+                    <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
+                </arguments>
+            </block>
```

Comment:
Why is this added here versus its own file? (Not arguing it should be its own file, I'm just curious)

Reply:
i placed it here because i believe this layout applies on every page, so regardless of what the base URL is, if the leadToken is present, the modal will work.

guidance puts it in `cms_index_index.xml` which would probably work for us too, but we haven't created that file yet, and i don't think this is too different. i didn't think too hard about this but we could definitely revisit if we decide it would be better elsewhere!